src/components/Map/style.css

```diff
@@ -1,18 +1,41 @@
-.mzw-map__container {
-  height: 86%;
-  
-}
+@import '../../vars.css';
 
-.mzw-map__fetching {
-  display: flex;
-  justify-content: center;
-  margin-top: 2.5rem;
+.mzw-map-container {
+  height: 75vh;
 }
 
 .mzw-map {
   height: 100%;
 }
 
-.mzw-map--fetching {
-  display: none;
+.mzw-map-marker {
+  display: block;
+  margin-top: 5px;
+  min-width: 250px;
+  font-size: 12px;
+  font-family: 'Source Sans Pro', 'HelveticaNeue', 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
```

Comment:
This should just be source sans and system ui as cssnext will transform it to the above

Reply:
system-ui turns into helvetica etc?